discoverable-client/src/main/java/com/ca/mfaas/client/api/PetControllerExceptionHandler.java

```diff
@@ -32,39 +32,38 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Collectors;
 
 /**
  * This class creates responses for exceptional behavior of the PetController
  */
 @ControllerAdvice(assignableTypes = {PetController.class})
 @Order(Ordered.HIGHEST_PRECEDENCE)
 public class PetControllerExceptionHandler {
-    private static final Logger LOGGER = LoggerFactory.getLogger(PetControllerExceptionHandler.class);
 
-    private final ErrorService errorService;
+    private final MessageService messageService;
 
     /**
      * Constructor for {@link PetControllerExceptionHandler}.
-     * @param errorService service for creation {@link ApiMessage} by key and list of parameters.
+     * @param messageService service for creation {@link Message} by key and list of parameters.
```

Comment:
not comprehensible. so, some service is used/employed to create smth ({@link Message}?)? you can create something 'with', not 'by'. Or you can use 'smth' to create something else. Given this, what's the point you try to deliver here?